addons/hr/static/tests/helpers/model_definitions_setup.js

```diff
@@ -2,7 +2,7 @@
 
 import { addFakeModel, addModelNamesToFetch } from '@bus/../tests/helpers/model_definitions_helpers';
 
-addModelNamesToFetch(['hr.employee', 'hr.employee.public', 'hr.department']);
+addModelNamesToFetch(['hr.employee', 'hr.employee.public', 'hr.department', "hr.work.location"]);
```

Comment:
While modifying the line, you can correct the single quotes as well :)

Reply:
actually, considering it's in the master branch. Can I just use the pre-commit tooling to correct the files changed? now I have to bypass it since the file avatar_card_popover_patch.js is wrongly formatted

Reply:
It will end up with something like this, as the commit. If you think it is too much, I can revert the changes

Reply:
LGTM, I'm all down for that kind of change :ok_hand: Could you however specify in the commit message that it involves linting changes (or if it's no to bothersome, a separate commit), so that people relying on the blame to navigate changes are not too bothered by that change.
Thanks!